### PR TITLE
Updated Opalites

### PR DESCRIPTION
**Pull request by Reveraine**
I added names, some descriptions, and fixed up the Sea-Devil traits to make more sense/added flavor text.

### CODE REVIEW

File: CharacterCreation/01_04_Species_Opaleites.txt [hunk 1]
```diff
@@ -117,12 +122,12 @@ Appearance:
    
 == Available Traits ==
 
-Colorful
-   People like you, they naturally flock toward your brilliant scales.
-   You are the perfect disraction, bringing in the attention of those
+Colorful:
+   People are attracted by colors, they naturally flock toward brilliant scales.
+   They are the perfect disraction, bringing in the attention of those
```
Comment (rosebrallier) [inline CharacterCreation/01_04_Species_Opaleites.txt]: *distraction

File: CharacterCreation/01_04_Species_Opaleites.txt [hunk 2]
```diff
@@ -117,12 +122,12 @@ Appearance:
...
    in the area. 
    [Gain advantage on a Performance[CHA] check while attempting to distract
-   a group(or single) of individuals. Mechanical species and blind/eyeless
+   a either a group or single individual. Mechanical species and blind/eyeless
```
Comment (rosebrallier) [inline CharacterCreation/01_04_Species_Opaleites.txt]: "a either a" --> either a

File: CharacterCreation/01_04_Species_Opaleites.txt [hunk 3]
```diff
@@ -134,7 +139,7 @@ Home System: Trioch IV
 Stats: +1 STR, +1 FOR
 
 Appearance:
-   Thick, cartilaginous scales cover 
+   Thick, cartilaginous scales cover a wide skeletal structure.
```
Comment (rosebrallier) [inline CharacterCreation/01_04_Species_Opaleites.txt]: Not sure if "wide" is the best adjective here. m/b extensive, thick, or sturdy?

File: CharacterCreation/01_04_Species_Opaleites.txt [hunk 4]
```diff
@@ -155,21 +160,40 @@ Condrich does wear armor, they get an Armor Strength bonus of 20 to whichever
...
 ===== Sea Devils =====
+
     
 Size: Medium
 Home System: Trioch IV
 
 Stats: +2 DEX
 
 Appearance:
+    Similar in size to the Mendars, the Sea Devils, or Samek in the Opaleite
```
Comment (rosebrallier) [inline CharacterCreation/01_04_Species_Opaleites.txt]: *"to Mendars" and "Sea Devils"? Not sure if the "the" is needed/helpful

File: CharacterCreation/01_04_Species_Opaleites.txt [hunk 5]
```diff
@@ -155,21 +160,40 @@ Condrich does wear armor, they get an Armor Strength bonus of 20 to whichever
...
+tongue, are named for their inherenet ability to bioluminesce. 
 
 == Available Traits ==
 
-Bioluminesce
-
-   [
+Light Dance:
+  
+    Sea Devils can harness their ability to bioluminesce to distract and
```
Comment (rosebrallier) [inline CharacterCreation/01_04_Species_Opaleites.txt:178]: "bioluminesce" doesn't have any Google results- I don't think it exists as a word yet. We may want to edit to "their bioluminescence," though creating the verb form certainly doesn't hurt comprehension for me.

File: CharacterCreation/01_04_Species_Opaleites.txt [hunk 6]
```diff
@@ -155,21 +160,40 @@ Condrich does wear armor, they get an Armor Strength bonus of 20 to whichever
...
-
-   [
+Light Dance:
+  
+    Sea Devils can harness their ability to bioluminesce to distract and 
+confuse enemies. Used in conjunction with darkness and quick movement, an
+experienced individual can alternate lighting scales from tip to tail to 
+misdirect an enemy.
+
+   Under low-light or darker, may use this ability while evading enemy
```
Comment (rosebrallier) [inline CharacterCreation/01_04_Species_Opaleites.txt]: *"or darker conditions, you may"

File: CharacterCreation/01_04_Species_Opaleites.txt [hunk 7]
```diff
@@ -155,21 +160,40 @@ Condrich does wear armor, they get an Armor Strength bonus of 20 to whichever
...
+fire to give you a +1 to hit. Enemies may roll Awareness vs 60 to see
```
Comment (rosebrallier) [inline CharacterCreation/01_04_Species_Opaleites.txt]: *"+1 to be hit"?

Comment (rosebrallier) [inline CharacterCreation/01_04_Species_Opaleites.txt]: "roll a DC 60 Awareness Save"

File: CharacterCreation/01_04_Species_Opaleites.txt [hunk 8]
```diff
@@ -155,21 +160,40 @@ Condrich does wear armor, they get an Armor Strength bonus of 20 to whichever
...
+throught the deception. May also be used to attempt to distract an
+individual away from your teammates; enemy awareness vs. Acro[DEX] or Alth[DEX].
```
Comment (rosebrallier) [inline CharacterCreation/01_04_Species_Opaleites.txt]: *Awareness vs. Acrobatics (Dex) or Athletics (Dex)

File: CharacterCreation/01_04_Species_Opaleites.txt [hunk 9]
```diff
@@ -155,21 +160,40 @@ Condrich does wear armor, they get an Armor Strength bonus of 20 to whichever
...
    
-Deep Glow
+Deep Glow:
 
-   [Light a 10m area around you with a dim light. +1 to enemies
-   trying to hit you]
+    Even those with Low-light vision cannot see in total darkness without
```
Comment (rosebrallier) [inline CharacterCreation/01_04_Species_Opaleites.txt]: *low-light

File: CharacterCreation/01_04_Species_Opaleites.txt [hunk 10]
```diff
@@ -155,21 +160,40 @@ Condrich does wear armor, they get an Armor Strength bonus of 20 to whichever
...
+aid. A Sea Devil may concentrate on their bioluminescent scales, brightening
+them to a leve that will allow others to see better in the darkness.
```
Comment (rosebrallier) [inline CharacterCreation/01_04_Species_Opaleites.txt]: *level

File: CharacterCreation/01_04_Species_Opaleites.txt [hunk 11]
```diff
@@ -155,21 +160,40 @@ Condrich does wear armor, they get an Armor Strength bonus of 20 to whichever
...
+    
+    The area 5m around you is increased two light levels, an additional
```
Comment (rosebrallier) [inline CharacterCreation/01_04_Species_Opaleites.txt:195]: "The light level of the area 5m around you is increased by two, and an..."

File: CharacterCreation/01_04_Species_Opaleites.txt [hunk 12]
```diff
@@ -155,21 +160,40 @@ Condrich does wear armor, they get an Armor Strength bonus of 20 to whichever
...
+5m is increased by a single light level. +1 to enemies trying to hit you.
```
Comment (rosebrallier) [inline CharacterCreation/01_04_Species_Opaleites.txt]: "Enemies attacking you gain a +1 bonus to hit"

File: CharacterCreation/01_04_Species_Opaleites.txt [hunk 13]
```diff
@@ -155,21 +160,40 @@ Condrich does wear armor, they get an Armor Strength bonus of 20 to whichever
...
-   trying to hit you]
+    Even those with Low-light vision cannot see in total darkness without
+aid. A Sea Devil may concentrate on their bioluminescent scales, brightening
+them to a leve that will allow others to see better in the darkness.
+    
+    The area 5m around you is increased two light levels, an additional
+5m is increased by a single light level. +1 to enemies trying to hit you. 
+This takes a great deal of energy; it will take twice as long to perform
+actions in combat while this ability is active, and any injury will end
+the ability. Can be used up to 10 minutes before needing a short rest;
```
Comment (rosebrallier) [inline CharacterCreation/01_04_Species_Opaleites.txt:199]: *"used for...requiring"

File: CharacterCreation/01_04_Species_Opaleites.txt [hunk 14]
```diff
@@ -117,12 +122,12 @@ Appearance:
...
 == Available Traits ==
 
-Colorful
-   People like you, they naturally flock toward your brilliant scales.
-   You are the perfect disraction, bringing in the attention of those
+Colorful:
+   People are attracted by colors, they naturally flock toward brilliant scales.
+   They are the perfect disraction, bringing in the attention of those
    in the area. 
    [Gain advantage on a Performance[CHA] check while attempting to distract
```
Comment (rosebrallier) [inline CharacterCreation/01_04_Species_Opaleites.txt]: While we're here, "Performance (Cha)"

**Review by rosebrallier — APPROVED** [review]
Shiny! I left some grammar, sentence structure and diction notes.

File: CharacterCreation/01_04_Species_Opaleites.txt (hunk 11 above)
Comment (bleehu) [reply CharacterCreation/01_04_Species_Opaleites.txt:195]: Also, what are light levels? This was covered in @dfuchs2 's PR #800

**Review by bleehu — APPROVED** [review]
One note on merge-ability, other than that, shipit!

**Comment by bleehu** [comment]
linking Issue #297

**Pull request merged**